deploy/kubernetes/releases/csi-digitalocean-dev.yaml

```diff
@@ -374,3 +235,43 @@ spec:
         - name: device-dir
           hostPath:
             path: /dev
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: csi-do-node-sa
+  namespace: kube-system
+
+---
+
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: driver-registrar-binding
```

Comment:
from this whole renaming, the only 2 resources without `do` in their names are:
- `driver-registrar-binding`
- `driver-registrar-role`

are these ❄️ special ? otherwise might as well change them too

Reply:
They don't, changing them 👍